.github/scripts/contrib.sh

```diff
@@ -9,7 +9,7 @@ git log --pretty=format:' - %ae' CONTRIBUTORS |sort |uniq |sort -f | sed "s/@/$a
 
 headAuthor=$(git log -1 --pretty=format:'%ae')
 authorAt=$(echo "$headAuthor" | sed "s/@/$antispam/")
-if git log --pretty=format:'%ae' CONTRIBUTORS | grep -qi "$headAuthor"; then
+if git log --pretty=format:'%ae' CONTRIBUTORS | grep -i "$headAuthor" > /dev/null; then
```

Comment:
why did you change this?

Reply:
The CONTRIBUTOR check was failing otherwise:

https://github.com/ankitects/anki/runs/468917834

(I'll admit I don't really understand why -- I feel like it should work. Something about `pipefail` made the old version fail).

Reply:
Ah, that makes sense thanks - grep will be closing the input as soon as it finds a line, which causes git log to lose its pipe.